public/pages/workflow_detail/agentic_search/test_flow/search_query.tsx

```diff
@@ -264,13 +369,23 @@ export function SearchQuery(props: SearchQueryProps) {
                 onBlur={handleAdvancedQueryChange}
                 editorHeight="200px"
                 isInvalid={!!jsonError}
+                helpText={
+                  <EuiText size="xs">
+                    For more information on configuring
+                    <EuiCode transparentBackground>agentic</EuiCode>queries,
+                    check out the{' '}
+                    <EuiLink href={AGENTIC_QUERY_DSL_DOCS_LINK} target="_blank">
```

Comment:
Can also add `rel = "noopener noreferrer"` for better security

Reply:
makes sense. added